network/burrow/simplenetwork/burrow-simple.json

```diff
@@ -14,6 +14,7 @@
       }
     }
   },
+  "contract": {"id": "simple", "path": "src/contract/burrow/simple/simple.bin", "language":"solidity", "version": "v0", "channel": "mychannel"},
```

Comment:
Only the `path` property is used in the code. These properties should be Burrow-specific. So whatever information you need to deploy the contract should be present, but the other properties should be removed to avoid confusion.